modules/services/window-managers/i3-sway/lib/options.nix

```
@@ -819,4 +819,14 @@ in {
     description = "Default launcher to use.";
     example = "bemenu-run";
   };
+
+  defaultWorkspace = mkOption {
+    type = types.nullOr types.str;
+    default = null;
+    description = ''
+      The default workspace to show when ${if isSway "sway" else "i3"} is launched.
```

Sorry, I gave you the wrong format. I think it should be:
```suggestion
      The default workspace to show when ${if isSway then "sway" else "i3"} is launched.
```

---

✅